pwa/src/templates/organizationDetail/OrganizationDetailTemplate.tsx

```diff
@@ -33,9 +33,22 @@ export const OrganizationDetailTemplate: React.FC<OrganizationDetailTemplateProp
           <Heading1>Gemeente Rotterdam</Heading1>
 
           <LeadParagraph>
-            Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas faucibus mollis interdum. Duis
-            mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Vivamus sagittis
-            lacus vel augue laoreet rutrum faucibus dolor auctor.
+            <div className={styles.quote}>
```

Comment:
Couple of notes here:

- Do we (will we) actually get the data for a quote item from the backend? If not, this might work in this dummy-data scenario but it will not work with real data;
- Could we use our `<QuoteWrapper />` component here? Instead of making this custom element.